src/Console/ChromeDriverCommand.php

```diff
@@ -59,6 +59,8 @@ class ChromeDriverCommand extends Command
     protected $slugs = [
         'linux' => 'linux64',
         'mac' => 'mac64',
+        'mac-intel' => 'mac64',
+        'mac-arm' => 'mac64_m1',
```

Comment:
`mac` key will remain for fallback in case architecture value doesn't match `x86_64` or  `arm64`.